dhcp-relay/dhcp_kern_xdp.c

```diff
@@ -166,27 +258,18 @@ static __always_inline int calc_ip_csum(struct iphdr *oldip, struct iphdr *ip,
  */
 //static __u8 buf[static_offset + VLAN_MAX_DEPTH * sizeof (struct vlan_hdr)];
 
-#define bpf_printk(fmt, ...)                                    \
-({                                                              \
-        char ____fmt[] = fmt;                                   \
-        bpf_trace_printk(____fmt, sizeof(____fmt),              \
-                         ##__VA_ARGS__);                        \
-})
-
 /* XDP program for parsing the DHCP packet and inserting the option 82*/
 SEC(XDP_PROG_SEC)
 int xdp_dhcp_relay(struct xdp_md *ctx) {
 
-	bpf_printk("\n");
-
 	/* Tail extend packet */
 	int res = bpf_xdp_adjust_tail(ctx, delta);
 	if (res != 0) {
 		bpf_printk("Cannot tail extend packet, delta %i - error code %i", delta, res);
-		return XDP_ABORTED;
+		return XDP_PASS;
```

Comment:
This is a packet we figured we should handle, but that failed; so what is XDP_PASS supposed to do with it? Better to drop...